pkg/minishift/openshift/openshift.go

```diff
@@ -69,11 +69,11 @@ func (t *OpenShiftPatchTarget) localConfigFilePath() (string, error) {
 }
 
 func RestartOpenShift(commander docker.DockerCommander) (bool, error) {
-	fmt.Println("Restarting OpenShift")
 	ok, err := commander.Restart(constants.OpenshiftContainerName)
 	if err != nil {
 		return false, err
 	}
+	fmt.Println("OpenShift restarted successfully")
```

Comment:
I would suggest putting the success message in cmd/openshift/restart.go as ideally, the package should only return to cmd and cmd should success or failure log to console. I know we are not consistent in the code about it, but I think that's the right thing to do.

Reply:
agree with @LalatenduMohanty to move this out of the code block. Since we already retrurn a`bool` to indicate the state, we can easily handle this outside this function.